doc/man/lttng-enable-event.1.txt

```diff
@@ -411,8 +411,8 @@ Dynamic user space probes
 With the option:--userspace-probe option, you can instrument function
 entries of any user space binary (application or library) using either
 an available symbol name or a SystemTap Statically Defined Tracepoint
-(SDT, a DTrace-style marker) probe's provider and probe names. As of
-this version, only SDT probes that are :not: surrounded by a reference
+(USDT, a DTrace-style marker) probe's provider and probe names. As of
```

Comment:
Could you replace _Statically Defined Tracepoint_ above with _User Statically Defined Tracing_? I made a mistake, the _T_ in SDT is _Tracing_.